lms/djangoapps/teams/static/teams/js/views/edit_team_members.js

```diff
@@ -0,0 +1,103 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
+            'teams/js/models/team',
+            'teams/js/views/team_utils',
+            'common/js/components/utils/view_utils',
+            'text!teams/templates/edit-team-member.underscore',
+            'text!teams/templates/date.underscore'
+    ],
+        function (Backbone, $, _, gettext, TeamModel, TeamUtils, ViewUtils, editTeamMemberTemplate, dateTemplate) {
+            return Backbone.View.extend({
+                dateTemplate: _.template(dateTemplate),
+                teamMemberTemplate: _.template(editTeamMemberTemplate),
+                errorMessage: gettext("An error occurred while removing the member from the team. Try again."),
+
+                events: {
+                    'click .action-remove-member': 'removeMember'
+                },
+
+                initialize: function(options) {
+                    this.teamMembershipDetailUrl = options.context.teamMembershipDetailUrl;
+                    // The URL ends with team_id,request_username. We want to replace
+                    // the last occurrence of team_id with the actual team_id, and remove request_username
+                    // as the actual user to be removed from the team will be added on before calling DELETE.
+                    this.teamMembershipDetailUrl = this.teamMembershipDetailUrl.substring(
```

Comment:
This might be made a little cleaner with `_.string.words`, but it doesn't matter much.